recipes/puppies/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "exo_puppies" %}
+{% set version = "0.2.0a1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/exo_puppies-{{ version }}.tar.gz
+  sha256: d761731eec8a2c84825c294ebda4587cbb2e944d6206fdfbf1ff65f9d6b9be5f
+
+build:
+  number: 0
+  skip: true  # [win]
+  skip: true  # [py<35]
+  entry_points:
+    - pup = puppies.__main__:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - numpy
+    - pip
+    - python
+  run:
+    - astropy >=3.1
+    - matplotlib-base >=1.3.1
+    - mc3 >=3.0.6
+    - numpy >=1.8.1
+    - python
+    - scipy >=0.13.3
+    - {{ pin_compatible('numpy') }}
+
+test:
+  imports:
+    - puppies
```

Please add this too:

```suggestion
    - puppies
  commands:
     - pip check  
  requires:    
    - pip
```